### DISCUSSION TITLE
How to redirect a POST request to Mockoon

### DISCUSSION THREAD

**Discussion by ionuttibi** [mockoon/mockoon#984] [Help]
I have a post request to URL `http://localhost:4000/`  with a body,  and i want it to get intercepted by Mockoon which runs at `http://localhost:4001/`

How can i do this? The proxy setting works exactly in reverse, redirecting from Mockoon to original server

**Comment by 255kb** [comment]
Mockoon needs to sit between your API and the app consuming it. So, you need to connect your app to Mockoon directly using port 4001. Mockoon will not intercept the request. 
One solution could be to setup port forwarding between the two ports on your computer. But you would have to change your API port I guess.
